modules/bismark_methXtract.cfmod

```diff
@@ -130,8 +123,8 @@ if($files && scalar(@$files) > 0){
 		
 		# Find if PE or SE from input BAM file
 		if(CF::Helpers::is_bam_paired_end($file)){
-			
-			my $command = "bismark_methylation_extractor $ignore_r1 $ignore_r2 --bedGraph --counts $buffer --cytosine_report --gzip -p --no_overlap --report $gw_cov $file";
+			# using 4 threads for the exctraction
```

Comment:
This should use the number of cores assigned to the module (`$cores`). Any complaints if I change it to that?

Need to change line 41 above to always return 4 instead of 1 to do what you're trying to do here. As it is, the job scheduler will only give 1 core to the module.